CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+12. Wait for the pull request to be reviewed by a maintainer.
+13. Introduce changes to the pull request if the reviewing maintainer recommends them.
+14. Celebrate your success after your pull request is merged!
+
+## Feature requests
+
+## Code reviews
+
+All submissions, including submissions by project members, require review.
+
+Score uses GitHub pull requests for this purpose. Consult [GitHub Help](https://help.github.com/articles/about1.pull1.requests/) for more information on using pull requests.
+
+The general workflow for code contributions:
+
+1. Submit/find an issue in this repository.
+1. Clone the relevant repo.
+1. Make your code change.
+1. Write tests and update docs.
+1. Build and test locally.
+1. Submit a pull request.
+1. Iterate as needed.
+1. Your PR will be approved and merged.
+
+### **Where can I go for help?**
+
+If you need help, you can create an issue.
+
+### Reporting bugs
+
+#### Before submitting an issue
+
+A good bug report shouldn't leave others needing to chase you up for more information. Therefore, we ask you to investigate carefully, collect information and describe the issue in detail in your report. Please complete the following steps in advance to help us fix any potential bug as fast as possible.
+
+- Make sure that you are using the latest version.
+- Determine if your bug is really a bug and not an error on your side for example using incompatible environment components/versions (Make sure that you have read the [documentation](https://github.com/score-dev/docs). If you are looking for support.
+- To see if other users have experienced (and potentially already solved) the same issue you are having, check if there is not already a bug report existing for your bug or error in the [bug tracker](https://github.com/score-dev/demo-repositoryissues?q=label%3Abug).
+- Also make sure to search the internet (including Stack Overflow) to see if users outside the GitHub community have discussed the issue.
+- Collect information about the bug:
+  - Stack trace (Traceback).
+  - OS, Platform and Version (Windows, Linux, macOS, x86, ARM).
+  - Version of the interpreter, compiler, SDK, runtime environment, package manager, depending on what seems relevant.
+  - Possibly your input and the output.
+  - Can you reliably reproduce the issue?
+
+### **What does the Code of Conduct mean for me?**
+
+Our [Code of Conduct](CODE_OF_CONDUCT.md) means that you are responsible for treating everyone on the project with respect and courtesy, regardless of their identity. If you are the victim of any inappropriate behavior or comments as described in our Code of Conduct, we are here for you and will do the best to ensure that the abuser is reprimanded appropriately, per our code.
+
+## Community Guidelines
+
+This project follows the [CNCF Community Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
```

Comment:
I think the idea was to use our code of conduct for now and once we applied to CNCF sandbox we'll reference theirs instead.  Would remove this for now.